admin/sections/config/chat.tsx

```diff
@@ -57,9 +62,11 @@ export const ChatConfig = ({
           </FormItem>
         )}
       />
```

Comment:
Idea is to never show the submit button and submit automatically on focus lost

Reply:
I still prefer to click Update to submit the changes instead of doing it automatically, because this is admin setting which affects the entire app.

Reply:
on the other hand we also already automatically "submit" if we delete or upload a file

Reply:
>I still prefer to click Update to submit the changes instead of doing it automatically
is that still the case - what do you suggest @leehuwuj ?